.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: build
```

Comment:
Thinking out loud: GitHub Actions seems fine for now because this shouldn't run long but it looks like it can cost us money. We may want to consider a free service – e.g. travis – or switching to TaskCluster that we probably have more control over the costs of. Maybe? Dunno. Again, seems fine for now.

Reply:
My understanding is that Github Actions jobs are free for any public repo (based on this page: https://github.com/features/actions). We could use Travis, but I've found that its reliability isn't great, especially for macOS builders (eventually, I'd like to test all platforms so we know that FNPRMS can be used locally by any developer). TaskCluster would be an option, but I'm not sure how freely we can add new tasks to it; I see a lot of mozilla-mobile projects use Travis instead...

Reply:
Oh, I totally glossed over that public/private distinction. WFM!